src/include/bootloader/routines/common_helps.rb

```diff
@@ -25,7 +25,7 @@ def initialize_bootloader_routines_common_helps(_include_target)
         )
       }
 
-      @common_descriptions = { "timeout" => _("&Timeout in Seconds") }
+      @common_descriptions = { "timeout" =>  }
```

Comment:
thanks for catch, I actually plan to remove whole file. I just removing part by part, to not forget anything